src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.RuntimeConfig.targets

```diff
@@ -38,7 +38,17 @@ See: https://github.com/dotnet/runtime/blob/b13715b6984889a709ba29ea8a1961db469f
         Trim="true" />
     <!-- https://github.com/dotnet/runtime/blob/211cdd011f19a51b7092d8365e11e774a8280afb/src/libraries/System.Private.CoreLib/src/System/LocalAppContextSwitches.cs#L52 -->
     <RuntimeHostConfigurationOption Include="Switch.System.Reflection.ForceInterpretedInvoke"
-        Value="$(_SystemReflectionForceInterpretedInvoke)"
+        Value="$(AndroidAvoidEmitForPerformance)"
```

Comment:
This gives me some pause: Do we want/need/should we allow `Switch.System.Reflection.ForceInterpretedInvoke` and `Microsoft.Extensions.DependencyInjection.DisableDynamicEngine` to be handled separately?

`$(_SystemDiagnosticsMetricsMeterIsSupported)` allows `System.Diagnostics.Metrics.Meter.IsSupported` to be handled separately.

I wonder if we should leave this line unchanged -- continue using `Value="$(_SystemReflectionForceInterpretedInvoke)"` -- ***and also*** have the default values of `$(_SystemReflectionForceInterpretedInvoke)` and `$(_SystemReflectionForceInterpretedInvoke)` be the opposite of `$(AndroidAvoidEmitForPerformance)`.

(Aside: how concerned should I be that one of these switches has a `Switch.` prefix, while the other doesn't?)

Reply:
I think the reason to put both under a single property is simplicity. I can't really think of a good reason to toggle them at all -- let alone independently.

> (Aside: how concerned should I be that one of these switches has a Switch. prefix, while the other doesn't?)

`Switch.` seems weird to me, too, but they have a mix of both in this file:

https://github.com/dotnet/runtime/blob/a5b75b8179d7db4ff720e19233d2b99fd15b94f0/src/libraries/System.Private.CoreLib/src/System/LocalAppContextSwitches.cs

I think it depends on which team implemented it...